docs/bank-connect/rest-api.md

```diff
@@ -168,6 +169,7 @@ The response has the following fields:
     | account_category | string | account category, can be `individual` or `corporate` |
     | account_number | string | account number |
     | credit_limit   | Integer |  limit up to which a company can withdraw from the working capital limit sanctioned |
+    | last_updated | Timestamp |  Timestamp when the account was last updated|
```

Comment:
1. specify the format of timestamp here
2. specify what last updated means in context of account, as in the account where last statement was uploaded to

Reply:
resolved

Reply:
put the explanation in this table itself instead of in the bullet points